ui/qt-lib/ariadne-qt-lib.cabal

```diff
@@ -25,7 +25,8 @@ library
   hs-source-dirs:
       src
   build-depends:
-      ariadne-core
+      ariadne-cardano
```

Comment:
This is definitely wrong, because UI lib _must not_ depend on concrete currency backend.